integration-cli/docker_api_logs_test.go

```diff
@@ -82,3 +86,96 @@ func (s *DockerSuite) TestLogsAPIContainerNotFound(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusNotFound)
 }
+
+func (s *DockerSuite) TestLogsAPIUntilFutureFollow(c *check.C) {
```

Comment:
Maybe it should - I wasn't quite sure how to do that. Would I just need the `return` here https://github.com/moby/moby/pull/32914#discussion-diff-130631641R126 as you indicated?